premium/ecommerce/gdpr-send-to-mailchimp.php

```diff
@@ -5,38 +5,99 @@
  */
 
 /**
- * Send a request to mailchimp to check if member has status subscribed or pending
+ * Check if a list member has status pending or subscribed based on there email
```

Comment:
Can we add here that it uses the Mailchimp List ID from the **MC4WP > E-commerce** settings page?